recipes/21cmFAST/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "21cmFAST" %}
+{% set version = "3.0.2.dev0" %}
```

Comment:
Is there a non-development version of the package available?

Reply:
Agh! I thought I had updated that. It's good to go now.